### ISSUE TITLE
EnsureThat not loading properly?

### ISSUE THREAD

**Issue by ashelley** [danielwertheim/requester#7]
Hello, in my project I am able to use Ensure.That no problem.  However when I use "When" from Requester.Validation I get the following error:

```
Result Message: System.IO.FileNotFoundException : Could not load file or assembly 'EnsureThat, Version=4.0.0.26, Culture=neutral, PublicKeyToken=null' or one of its dependencies. The system cannot find the file specified.
Result StackTrace:  
at Requester.Validation.When.DoRequest(HttpMethod method, String url, Action`1 configurer, Func`2 contentFn)
   at Requester.Validation.When.Put(String url, Action`1 configurer)
```

I'm not sure what the specific issue is.  Currently, to work around this issue I have simply copy and pasted the source code for 'When.cs' into my project and called it 'MyWhen'... I removed the c# 6.0 specific syntax (usage of the null conditional operator) and everything seems to work.

Do you know why this assembly might not load in the specific use case of using the Requester.Validation.When methods?

as an aside the documentation for requester has a small error:

```
Debug.WriteLine(response.ToStringDebugVersion(includeContent: true));
```

includeContent parameter does not seem to be valid any more.

Edit: Screen shot of exception

![image](https://cloud.githubusercontent.com/assets/3195045/18528986/1edc0306-7a80-11e6-91ec-8fad93759c4d.png)

**Comment by danielwertheim** [comment]
Sorry for the really late answer.

No, I don't know why as they are taking the exact same dependencies etc. Will look into it (if I can reproduce)

**Comment by ashelley** [comment]
@danielwertheim i had an 'aha' moment and I thought I could solve this problem but it ended up being a dead end.  Here is a project to reproduce the problem:

https://github.com/ashelley/RequesterRepro

Step 1) clone repo and open solution with visual studio 2015
Step 2) Run tests using visual studio test runner

Expected results: Test runs and can load Ensure.That internally

Actual results:

![image](https://cloud.githubusercontent.com/assets/3195045/19912109/6383b706-a055-11e6-9345-4698f2906931.png)

**Comment by danielwertheim** [comment]
Update the dependency https://github.com/danielwertheim/requester/blob/master/src/projects/Requester/packages.config#L3

And make a new release of Requester

**Comment by danielwertheim** [comment]
Forgot to associate checkin... this is now solved with latest release `v1.3.0`
